sidebars.js

```diff
@@ -13,6 +13,16 @@ module.exports = {
           "get-started/technical-concepts/consensus-staking",
           "get-started/technical-concepts/network-communication",
           "get-started/technical-concepts/governance-standards",
+          {
+            type: "category",
+            label: "Security & Key Management",
+            items: [
+              {
+                type: "autogenerated",
+                dirName: "get-started/security",
+              },
+            ],
```

Comment:
```suggestion
            type: "category",
            label: "Security & Key Management",
            items: [
              "get-started/security/air-gap",
              "get-started/security/secure-workflow",
              "get-started/security/frankenwallet",
            ],
```
The autogeneration from alphabetical order doesn't work here since "Secure Workflow" is applicable to everyone who uses the CLI.  "Frankenwallet" is also applicable to this set of users, but fewer people will be interested in pursuing it: so it should appear later/last.